lib/nodejs/supervisor.ex

```diff
@@ -15,10 +15,11 @@ defmodule NodeJS.Supervisor do
   ## Options
     * `:path` - (required) The path to your Node.js code's root directory.
     * `:pool_size` - (optional) The number of workers. Defaults to #{@default_pool_size}.
+    * `:name` - (optional) If there are multiple NodeJS supervisors, give then different names.
```

Comment:
We might want to fix this explanation a bit to state that this allows. Also, since you've added worker configuration then that should be documented.